tests/test_getDocumentIDs.py

```diff
@@ -0,0 +1,61 @@
+from c20_server import documentsIDs
+from dotenv import load_dotenv
+import os
+import requests
+import pytest
+import requests_mock
+import unittest
+from mock import patch
+import json
+
+URL = "https://api.data.gov:443/regulations/v3/documents.json"
+load_dotenv()
+api_key = os.getenv("KEY")
+dktid = "EPA-HQ-OAR-2011-0028"
+
+api_url = "{}?api_key={}&dktid={}".format(URL,api_key,dktid)
+
+
+class Tests(unittest.TestCase):
```

Comment:
we are using pytest, not unittest.

Reply:
okay, we are going to change it to pytest